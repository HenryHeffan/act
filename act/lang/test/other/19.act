bool a, b;

a = b;

{ a !== b : "Connected?" };

