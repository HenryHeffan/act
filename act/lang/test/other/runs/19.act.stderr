In expanding ::<Global>
Identifiers `a' and `b' are connected.
   message: Connected?
FATAL: Aborted execution on failed assertion
